base/range.jl

```diff
@@ -237,6 +237,7 @@ julia> linspace(1.3,2.9,9)
 1.3:0.2:2.9
 ```
 """
+linspace(start::T, stop::T, len::Real=50) where {T} = linspace(start, stop, Int(len))
```

Comment:
for the docstring's sake I think it would be better to switch these two lines and have the docstring attached to the more general signature